ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -2391,8 +2396,20 @@ struct TTableIndexInfo : public TSimpleRefCount<TTableIndexInfo> {
         return result;
     }
 
+    TString DescriptionToStr() const {
+        return std::visit([](const auto& v) {
+            if constexpr (requires { v.SerializeAsString(); }) {
```

Comment:
Can you explain why this check is necessary? I guess, we expect all variants of `SpecializedIndexDescription` to be of protobuf message type.

Reply:
We expect monostate for global secondary indexes, but I agree that is maybe better to check this instead